backend/app/legacy/controllers/info.py

```diff
@@ -82,3 +93,37 @@ def healthcheck() -> (Healthcheck, int):
         return healthcheck_status, 500
 
     return healthcheck_status, 200
+
+
+def sync_status() -> Tuple[SyncStatus, int]:
+    services, status = healthcheck()
+    blockchainEpoch = None
```

Comment:
we could change camelCase to snake_case for the sake of consistency in Python

Reply:
+1

Reply:
done